src/dodal/beamlines/training_rig.py

```diff
@@ -31,13 +29,18 @@
 set_log_beamline(BL)
 set_utils_beamline(BL)
 
-set_path_provider(
-    StaticVisitPathProvider(
-        BL,
-        Path("/exports/mybeamline/data/2025"),
-        client=LocalDirectoryServiceClient(),
-    )
-)
+# Leave the old path provider so it can be uncommented and used in the meantime while
+# we transition
+
+# set_path_provider(
+#     StaticVisitPathProvider(
+#         BL,
+#         Path("/exports/mybeamline/data/2025"),
+#         client=LocalDirectoryServiceClient(),
+#     )
+# )
```

Comment:
```suggestion
```

We can get it back from the git history

Reply:
From discussion, I think the idea is to revert this change and make blueapi call `set_path_provider(StartDocumentBasedPathProvider())` when given the proper config, but correct me if I'm remembering wrongly.

Reply:
I have put this back to use the old one, so we leave it in a working state. There is another issue for changing it to the new provider which depends on a numtracker issue to configure the training rigs.

https://github.com/DiamondLightSource/dodal/issues/1130

and 

https://github.com/DiamondLightSource/numtracker/issues/134